### DISCUSSION TITLE
stream only audio VOD twitch Channel to other platform

### DISCUSSION THREAD

**Discussion by ChristopheTdn** [sorayuki/obs-multi-rtmp#229] [Ideas]
You made a fantastic job  !!
OBS autorise to separate audio channel and VOD audio channel on **twitch** plateform.
i ask for an option to select the output audio (VOD channel) to the other plateform if possible. 
Thx

**Comment by IWantOriginalName** [comment]
I have same problem. Without plugin I can use VOD channel, but with plugin it does`t working. Twitch streamers need this setting! Please, @sorayuki add this🙏🏻!

**Comment by SubliminalsTV** [comment]
I third this! FYI you can still achieve the goal of multi-streaming and using the VOD track, but then you would have to use Twitch as your streaming platform inside obs, and then use obs multi rtmp for youtube. But then you lose the stream management features that YouTube needs. So you have to choose between VOD track and full YT integration.

**Comment by warlock-05** [comment]
I join the requests here, if the plugin has the ability to make a choice between OBS tracks that should go to the platform you are re-streaming, that would be very cool. Just checkboxes. I’ll just tell you what I personally encountered: I use sound alerts for various events, and if on Twitch (this is the main platform for me) I easily disable them for VOD, then after each stream on YouTube I have to check whether I have violated any copyrights with this alerts. It would be very convenient if you could simply exclude them. Thanks in advance for your work!

**Comment by MoraesGlider** [comment]
It would be nice to have this feature, that allows at least 2 different audio tracks, because we use one for live and one for VOD. Please consider this feature in the next updates!!!!
